js/src/lib/components/InferenceWidget/shared/WidgetExample.ts

```diff
@@ -0,0 +1,94 @@
+import type { TableData } from "./types";
+
+//#region outputs
+export type WidgetExampleOutputLabels = Array<{ label: string; score: number; }>;
```

Comment:
btw, I'm not sure if there was ever agreement between using:
`Array<T>` vs `T[]`

Reply:
not sure but here i copy/pasted this line from somewhere else